src/components/GetReport.jsx

```diff
@@ -1,76 +1,145 @@
 import React from "react";
 import { Formik, Form, Field } from "formik";
-import { GetResponseErrors } from "../utilities/CitysourcedResponseHelpers";
 import FormContainer from './FormContainer';
+import { SetFieldValues } from "../utilities/FormHelpers";
 import { GetReportByID } from '../services/authService';
+import '../css/inside-responsive.min.css';
+import '../css/citysourced-reporter.min.css';
+import SeButton from "./SeButton";
+import Note from './Note';
 import { Go, Routes } from "../Routing";
 
-const GetReport = props => {
-	const userGetReport = async (values) => {
+const GetReport = (props) => {
+    
+	const handleChange = changeEvent => {
+		SetFieldValues(props, {trackingNumber: changeEvent.target.value});
+	};
 
+	const userGetReport = async (trackingNumber) => {
+		const response = await GetReportByID(trackingNumber);
+		Go(props, Routes.ReportStatus, response)
+		return response;
+	}
 
-		try {
-			const response = await GetReportByID(values.ReportID);
-			if (response.data.ErrorsCount > 0) {
-				const errorsReturned = GetResponseErrors(response);
-				props.Field.ErrorMsg = errorsReturned;
-			}
-			else {
-				Go(props, Routes.AdditionalInformation);
-			}
+	const checkTrackingNumber = () =>{
+		const { trackingNumber } = props.values;
+	
+		if(RegExp(/^(ACCMP)/i).test(trackingNumber)){
```

Comment:
always think its good to give regexExp a variable so you can describe what they are doing.

Reply:
Changed these to semi generic named variables. Since we could use a different vendor the Accela or Citysourced I figured vendor would suffice. But honestly they are kind of craptacular names.

Reply:
this looks good, just one question commented on that line.